src/pytest_plugins/filler/filler.py

```diff
@@ -118,19 +117,12 @@ def default_html_report_file_path() -> str:
 def pytest_addoption(parser: pytest.Parser):
     """Add command-line options to pytest."""
     evm_group = parser.getgroup("evm", "Arguments defining evm executable behavior")
-    evm_group.addoption(
-        "--eels",
-        action="store_true",
-        dest="eels",
-        default=False,
-        help="Use eels t8n entry point",
-    )
     evm_group.addoption(
         "--evm-bin",
         action="store",
         dest="evm_bin",
         type=Path,
-        default="ethereum-spec-evm-resolver",
+        default=None,
```

Comment:
As per a previous [discussion](https://github.com/ethereum/execution-spec-tests/pull/1763#pullrequestreview-2941854830) we wanted to keep the resolver the default and have the need to explicitly specify that the EELS T8n entry point is to be used.